src/test/java/com/ea/restaurant/service/impl/utils/fixtures/UserFixture.java

```diff
@@ -0,0 +1,35 @@
+package com.ea.restaurant.service.impl.utils.fixtures;
+
+import com.ea.restaurant.constants.Roles;
+import com.ea.restaurant.constants.Status;
+import com.ea.restaurant.constants.UserType;
+import com.ea.restaurant.entities.User;
+import java.time.Instant;
+import java.util.Optional;
+
+public final class UserFixture {
+
+  public static User buildUser(
```

Comment:
You can use overload and create another function `buildUser()` but just sending the id of the user as parameter. Remember the concept of overload

Reply:
You should also add a `buildUser()` that accepts no parameters.

If you are going to add default values for the user then add them as `public static final` variables in this fixture

Reply:
Done.